WordPress/src/main/java/org/wordpress/android/ui/stats/StatsBarGraph.java

```diff
@@ -286,6 +286,22 @@ protected double getMinY() {
         return 0;
     }
 
+    // Make sure the highest number is always even, so the halfway mark is correctly balanced in the middle of the graph
+    // Also make sure to display a default value when there is no activity in the period.
+    @Override
+    protected double getMaxY() {
+        double maxY = super.getMaxY();
+        if (maxY == 0) {
+            return 10;
+        }
+
+        if (maxY % 2 == 0) {
```

Comment:
nitpick; could be replaced with `return maxY + (maxY % 2);` if you want to get tricky ;)